discovery-provider/plugins/network-monitoring/src/index.ts

```diff
@@ -19,6 +22,9 @@ const main = async () => {
 
     await closeDBConnection()
 
+    const tDelta = process.hrtime(t0)
+    await exportDuration(tDelta, run_id, totalJobDurationGauge)
```

Comment:
any reason to do this differently than the `gauge.startTimer()` per prom-client docs? https://github.com/siimon/prom-client#utility-functions

Reply:
we already have established patterns for this in CN codebase as well

Reply:
Nope, I must've missed this in the prom-client docs - I can go ahead and get that updated